### ISSUE TITLE
Non-english languages not showing up in catalog full item view

### ISSUE THREAD

**Issue by ulotrichous** [aadl/arborcat#65]
Seems to be a DVD thing as reported by a patron. English DVDs say english, but non-english DVDs don't specify. It looks like it's there in the marc.

Compare:
https://aadl.org/catalog/record/10365693
language: english appears under published.
marc header contains lang: eng

to:
https://aadl.org/catalog/record/10285407
shows no language under published.
marc header contains lang: fre

same bib on old:
http://old.aadl.org/catalog/record/1194558
lang: french shows up.

I assume this will impact language faceting too.

**Comment by taleon** [comment]
Nope, this won't be a problem with lang searches; the couch records have proper info. I still need to add better language processing for that metadata bar. It's not just a DVD problem, it's all materials. I have this comment in the metadata bar right now: {# Need to add better language processing here #}

Since this won't affect searching, I don't see it as a big priority. It's just a display issue I need to handle with lang code mapping to full language string.
